android/lib/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,17 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+		  package="com.cube.geofencing">
+	<uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
```

Comment:
`ACCESS_FINE_LOCATION` will use GPS (and so is more battery intensive), and `ACCESS_COARSE_LOCATION` uses e.g. wifi and cellular data. But actually it is up to system to choose the most appropriate location source, all the geofencing logic happens opaquely within the Google Play Services library, and is presumably designed to optimise battery and so on.

Saying all that, I should add both permissions here, not just one. Good spot :)